docs/user/user/pings/sent-by-glean.md

```diff
@@ -0,0 +1,59 @@
+# Pings sent by Glean
+
+If data collection is enabled, the Glean SDK provides a set of built-in pings that are assembled out of the box without any developer intervention.  The following is a list of these built-in pings:
+
+- [`baseline` ping](baseline.md): A small ping sent every time the application goes to foreground and background. Going to foreground also includes when the application starts.
+- [`deletion-request` ping](deletion_request.md): Sent when the user disables telemetry in order to request a deletion of their data.
+- [`events` ping](events.md): The default ping for events. Sent every time the application goes to background or a certain number of events is reached.
+- [`metrics` ping](metrics.md): The default ping for metrics. Sent approximately daily.
```

Comment:
I think it would be useful to signal here which pings are sent by which Glean. You could use a table as I did on the debugging docs changes PR, https://github.com/mozilla/glean/pull/1647/files#diff-17827c65c12bbe5289fda0778d9a980a4580160458127e0dfb0492860334b53bR60-R67